#17-Database-Diagram

bubriks opened the pull request.

bubriks committed:
#17 First draft

-Diagrams separated
-Models improved and added
-Documents written

bubriks committed:
#17 diagrams changed after meeting

RaidenRabit reviewed (commented):
- Please add numbers/"ids" to your headers (for example, Overview, should be 4.3.1. Overview)
- I don't know if Overview should be included, given that we don't have definitions for anything else
- perhaps highlighting the figure references would help when transcribing the things into the report (so we don't end up with fked numbers or forgoten things 🗡 
- there are some , and . missing, but we can fix that later, i guess
- the includede pics are a bit unreadable. maybe make them bigger?
- both diagrams need to be updated

bubriks committed:
#17 should solve most of the issues mentioned in review

bubriks committed:
#17 fixing diagrams

bubriks committed:
#17 fixing diagrams

RaidenRabit reviewed (commented):
Im ok with this, but @AdamBlazsek must review it as well 🖌

bubriks committed:
#17 not final changes

eer and database diagram still not 100% done
text needs review

bubriks committed:
#17 small diagram updates (naming)

bubriks committed:
#17 could be done

bubriks committed:
#17 small improvements

bubriks committed:
#17 Information in report and diagrams updated

bubriks committed:
#17 changes after meeting

RaidenRabit commented:
![image](https://user-images.githubusercontent.com/22789107/57890767-2d4cfd00-7839-11e9-8072-23730aea7d85.png)
is it really up to date? i mean in the script i think toDate is not PK anymore.
Also those are my thoughts on the NF forms for each table

@AdamBlazsek you should also check this stuff out :P

bubriks committed:
#17 needs work

bubriks committed:
17# everything should be fixed

bubriks committed:
#17 notice added

bubriks committed:
#17 should be up to date

changes after teachers request and database diagram is improved (variable types added)

bubriks committed:
Merge branch 'development' into #17-Database-Diagram

RaidenRabit reviewed (approved):
besides the underlying of pk's, nothing to add here
